### ISSUE TITLE
Mouse cursor on tablets and touch devices

### ISSUE THREAD

**Issue by Ahmed-Ali** [excalidraw/excalidraw#1778]
I noticed that touch devices still show mouse cursor.
It would be better if we tweak the UX for touch devices to feel more native

**Comment by vjeux** [comment]
Agreed, we should likely use a semi transparent filled circle instead of the arrow. Should be easy, we just need to send the fact that it's a touch event in the network.

**Comment by kolharsam** [comment]
@vjeux any work done on this issue yet?

**Comment by dwelle** [comment]
@kolharsam not that I'm aware of. Go for it!

**Comment by kolharsam** [comment]
@dwelle any pointers to help me get started?

**Comment by dwelle** [comment]
You'll want to add a prop to the payload object here:

https://github.com/excalidraw/excalidraw/blob/5970bb7ee95381d488b918e2995455685ae2b0b7/src/components/App.tsx#L1084-L1104

and modify the cursor rendering based on it in `renderScene`:

https://github.com/excalidraw/excalidraw/blob/5970bb7ee95381d488b918e2995455685ae2b0b7/src/renderer/renderScene.ts#L458

If anything, hop into our discord chat and ask for more help there https://t.co/jWJMO0IcUu?amp=1

**Comment by mtolmacs** [comment]
5 yo issue, closing it as stale, pls reopen if still applicable.
